schema/swagger.yml

```diff
@@ -14,20 +14,24 @@
 swagger: '2.0'
 
 info:
-  description: Lisk API Documentation
   title: Lisk API Documentation
-  version: '1.0.26'
+  description: Lisk API Documentation
+
+
+  version: '1.0.27'
   contact:
     email: admin@lisk.io
   license:
     name: GPL v3.0
     url: https://www.gnu.org/licenses/gpl-3.0.en.html
 
 # All paths relative to specified basePath
-# basePath: /v1
-
 basePath: /api
 
+# The host (name or ip) serving the API
+#host: 46.101.179.89:5000
+host: localhost:4000
+
```

Comment:
Don't make host and IP as static entry. First thing, port value can be different and configureable int he application, secondly instead of localhost one can restrict traffic only listen on local network IP.

Reply:
Alright, removed the host variable from `swagger.yml` for now.